src/manager/SafeBrowsingManager.js

```diff
@@ -0,0 +1,68 @@
+import fetch from 'node-fetch';
+
+export default class SafeBrowsingManager {
+  constructor() {
+    // Get Google Safe Browsing key
+    this.apiKey = process.env.GOOGLE_SAFE_BROWSING_API_KEY;
+  }
+
+  static checkApiKey() {
+    const instance = new SafeBrowsingManager();
+
+    if (!instance.apiKey) {
+      console.error(
+        `You need an environment variable for GOOGLE_SAFE_BROWSING_API_KEY. Please check slack for this key.
+    Quiting honeyclient now.`
+      );
+      process.exit();
+    }
+  }
+
+  async getMalwareMatches(URLs) {
+    // Make request to Google safe browsing
+    const safeBrowsingURL = `https://safebrowsing.googleapis.com/v4/threatMatches:find?key=${this.apiKey}`;
+    const requestURLList = URLs.map(entry => {
+      const container = {};
+      container['url'] = entry;
+      return container;
+    });
+    const request = {
+      client: {
+        clientId: '4950',
+        clientVersion: '0.0.1',
+      },
+      threatInfo: {
+        threatTypes: ['MALWARE'],
+        platformTypes: ['ANY_PLATFORM'],
```

Comment:
Could we expose the platform the user chose in the custom request options?

Reply:
Hmm... @alejx

Reply:
I don't think it's necessary...